ci/jobs/scripts/ssh-setup.sh

```diff
@@ -1,5 +1,2 @@
-#!/usr/bin/env bash
-set -euo pipefail
```

Comment:
Also this was required during some testing on the el7 master, and its removal did not cause the script to fail. What exactly about these settings ensures the script does not fail?